### ISSUE TITLE
Categories with pipe suffix

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#1826]
**Steps to reproduce:** 

1. With the app, find this picture of an interesting mushroom: https://commons.wikimedia.org/wiki/File:Cortinarius_vanduzerensis_134617.jpg
2. Notice how its categories are ended with `|・` or `|Mushroom`
3. Press the category. The app opens the category view activity, and shows zero media, evn though there should be at least that image present.

![screenshot_20180811-190736](https://user-images.githubusercontent.com/99590/44076143-167b1e8c-9fda-11e8-8050-e2cd94dba80d.png)

Source of the file:
```
{{Assessments|enwiki=1|enwiki-nom=Cortinarius vanduzerensis|featured=1}}
== {{int:filedesc}} ==
{{Information
|Description    ={{en|1=A fruit body of the North American mushroom '''''{{C|Cortinarius vanduzerensis}}''''' − A.H. Sm. & Trappe. 
*Specimen photographed in {{C|Salt Point State Park}}, Sonoma County, northern California, U.S.}}
|Source         ={{MushroomObserver|134617}}
|Author         =[http://mushroomobserver.org/observer/show_user/10 John Kirkpatrick (natashadak)]
|Date           =2011-02-15
|Permission     =
|other_versions =
}}

== {{int:license-header}} ==
{{cc-by-sa-3.0}}

[[Category:Cortinarius vanduzerensis]]
[[Category:Salt Point State Park|•]]
[[Category:Fungi of Sonoma County, California|Mushroom]]
{{LicenseReview|site=http://mushroomobserver.org/image/show_image/134617|user=Leoboudv|date=2017-12-22}}
```

I guess it is some sort of Commons convention? If anyone is familiar with such syntax please explain to us what it is, thanks!

I suppose we can safely remove the pipe and everything after it.

 **Commons app version:** 

Latest master

**Comment by VojtechDostal** [comment]
Yes that's just for alphabetical sorting of categorized media. You can safely remove everything after the pipe ("|") and the link should work

**Comment by domdomegg** [comment]
Fixed in #1873
